### ISSUE TITLE
robotframework-reporter-action@v2.5 fails with not a valid identifier

### ISSUE THREAD

**Issue by reubenmiller** [joonvena/robotframework-reporter-action#33]
PR https://github.com/joonvena/robotframework-reporter-action/pull/31 broke the bash shell script used to set the repository owner and name by accidentally using single quotes instead of double quotes resulting in the a failing of expansion of variables.

The problem would manifest when using `joonvena/robotframework-reporter-action@v2.5`, an example of the error is shown below:

```sh
Run joonvena/robotframework-reporter-action@v2.5
Run export OWNER='$(echo 'thin-edge/tedge-monit-setup' | awk -F / '{print $1}' | sed -e 's/:refs//')'
/home/runner/work/_temp/8a3a4c80-e9df-4cf9-aa36-e7e6386078e8.sh: line 1: export: `} | sed -e s/:refs//)': not a valid identifier
Error: Process completed with exit code 1.
```

Discovered on the following [build](https://github.com/thin-edge/tedge-monit-setup/actions/runs/9891571244/job/27322296697)

**Comment by reubenmiller** [comment]
I've created a PR with the fix #32

**Comment by joonvena** [comment]
I did now merge the fix and also released the change in same v2.5 tag.

**Comment by reubenmiller** [comment]
I've also been able to confirm that the new v2.5 version works as expected.
